app/helpers/patient_query_helper.rb

```diff
@@ -472,8 +472,15 @@ def advanced_filter_relative_date(patients, field, filter, tz_diff, type)
       timespan = filter[:value][:number].to_i.months if filter[:value][:unit] == 'months'
       return patients if timespan.nil?
 
-      timeframe = { after: (timespan.ago - tz_diff).beginning_of_day, before: local_current_time } if filter[:value][:when] == 'past'
```

Comment:
Why was the `.beginning_of_day` removed here and the `.end_of_day` removed on like 476 below?

Reply:
This was removed so that relative date fields with timestamps end at the current time on the bound and do not include the whole day.

Decision was documented here: https://mitrecorp.slack.com/archives/C01A6P94R9B/p1613682229290700